.github/workflows/release1.yaml

```diff
@@ -0,0 +1,58 @@
+name: Start release
+
+# There are a lot of builds for different languages that all need to agree on what the version is. To start a release, somebody
+# pushes a branch starting with "release-v". This workflow will then:
+# - Edit all the files that need to know the name of the release.
+# - Commit those files.
+# - Make a new PR for the release.
+# - Add a comment to the PR explaining what's going on.
+# When the release PR is merged, it triggers the second release workflow.
+
+on: create
+
+jobs:
+  kickoff:
+    if: github.event.ref_type == 'branch' && startsWith(github.event.ref, 'release-')
```

Comment:
Your comment says that the branch has to start with "release-v", but this only checks for "release-".  Which way were you intending?

Reply:
It'll work with any branch starting with `release-`, and it'll generate a new tag with that prefix removed. Our convention for releases it that they all look like `v#.#`. But somebody could try to make a release that's not numbered, like a special release for a particular customer. Maybe that's a bad idea though; I'm not sure.